setup-machine.sh

```diff
@@ -54,18 +54,20 @@ MODEL_NAME=${MODEL_NAMES[$CHOICE_INDEX]}
 echo "Excellent, let's set up ${CHOICE}."
 
 echo
-read -p "Is this image for QA where you want sudo privileges, the ability to record screengrabs, etc.? [y/N] " qa_image_flag
+read -p "Is this image for QA where you want sudo privileges, terminal access via TTY2, the ability to record screengrabs, etc.? [y/N] " qa_image_flag
 
 if [[ $qa_image_flag == 'y' || $qa_image_flag == 'Y' ]]; then
+    IS_QA_IMAGE=1
     VXADMIN_SUDO=1
```

Comment:
can we kill this var `VXADMIN_SUDO` now?

Reply:
Good catch! Will switch over to `IS_QA_IMAGE`